test/formats.js

```diff
@@ -0,0 +1,480 @@
+const tape = require('tape')
+const crypto = require('crypto')
+const SecretStack = require('secret-stack')
+const sleep = require('util').promisify(setTimeout)
+const pify = require('promisify-4loc')
+const u = require('./misc/util')
+
+const caps = require('ssb-caps')
+const rimraf = require('rimraf')
+const mkdirp = require('mkdirp')
+const ssbKeys = require('ssb-keys')
+const SSBURI = require('ssb-uri2')
+const bendyButt = require('ssb-bendy-butt')
+const { where, author, toPromise } = require('ssb-db2/operators')
+
+function createSsbServer() {
+  return SecretStack({ appKey: caps.shs })
+    .use(require('ssb-db2'))
+    .use(require('ssb-db2/compat/ebt'))
+    .use(require('../'))
+}
+
+const CONNECTION_TIMEOUT = 500 // ms
+const REPLICATION_TIMEOUT = 2 * CONNECTION_TIMEOUT
+
+const aliceDir = '/tmp/test-format-alice'
+rimraf.sync(aliceDir)
+mkdirp.sync(aliceDir)
+
+let alice = createSsbServer().call(null, {
+  path: aliceDir,
+  timeout: CONNECTION_TIMEOUT,
+  keys: u.keysFor('alice'),
+})
+
+const bobDir = '/tmp/test-format-bob'
+rimraf.sync(bobDir)
+mkdirp.sync(bobDir)
+
+let bob = createSsbServer().call(null, {
+  path: bobDir,
+  timeout: CONNECTION_TIMEOUT,
+  keys: u.keysFor('bob')
+})
+
+const bendyButtMethods = {
```

Comment:
I copied this object to my test now but that feels wrong..

Could we maybe expose this via `require('ssb-bendy-butt/ebt')` or something like this?

Reply:
or `ssb-ebt/bendy-butt` what ever you feel is best for maintenance. :)

Reply:
Right, that seems like a good idea. Similar with indexed thing. André is working on changing ssb-meta-feeds so we can get the information on what feeds are indexes and use that instead of adding the author to index among other things :)